addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -172,6 +173,13 @@ Wysiwyg.include({
         });
 
         this.$iframe.insertAfter(this.$editable);
+        this.$iframe.get(0).ownerDocument.addEventListener('click', (ev) => {
+            if (ev.target != this.$iframe.get(0) && !utils.getFirstAncestor(ev.target, (node) => node.id === "toolbar")){
+                if(this.$iframe.get(0) && this.$iframe.get(0).contentWindow) {
```

Comment:
_If_ we keep this:
```suggestion
            const iframe = this.$iframe.get(0)
            if (
                iframe  && ev.target !== iframe && iframe.contentWindow &&
                !utils.getFirstAncestor(ev.target, node => node.id === 'toolbar')
            ) {
```
Although I really don't like the idea of targeting the toolbar by id like this, and of dispatching a non-standard event for this. So I'd like to explore a little more if there's an alternative.

Also, this event is on `this.$iframe.get(0)` so why do you test if it exists? Is there a way for it not to exist anymore by the time we enter the callback?